client/my-sites/email/email-management/titan-control-panel-login-card/index.jsx

```diff
@@ -86,6 +119,42 @@ class TitanControlPanelLoginCard extends React.Component {
 			</div>
 		);
 	}
+
+	renderIframe() {
+		const { domain, translate } = this.props;
+		const translateArgs = {
+			args: {
+				domainName: domain.name,
+			},
+			comment: '%(domainName)s is a domain name, e.g. example.com',
+		};
+
+		return (
+			<div className="titan-control-panel-login-card">
+				<SectionHeader label={ translate( 'Titan Mail: %(domainName)s', translateArgs ) } />
+				<CompactCard>
+					{ this.state.iframeURL ? (
+						<iframe
+							title="Titan Control Panel"
+							src={ this.state.iframeURL }
+							width="100%"
+							height="650px"
+						/>
+					) : (
+						<div>Loading the control panel...</div>
```

Comment:
Is that a temporary string? If not, we should probably localize it (or use a placeholder).

Reply:
Bah, good point, will fix 🙇